src/pyOpenMS/pyopenms/plotting.py

```diff
@@ -1,3 +1,5 @@
+##
```

Comment:
What's this?
Can you add a nice python module docstring instead?

Reply:
Sorry, I forgot about these. I have tried out changes in the code directly in the script and I normally use CodeCell in PyCharm.

Reply:
No problem. A module documentation would still be very nice.
![Bildschirmfoto 2023-01-18 um 15 19 46](https://user-images.githubusercontent.com/8102638/213195215-1ac36f7c-c3de-4aa3-89af-a2801fd5ac7a.png)

Reply:
Do we have an example somewhere for that?